functions.php

```diff
@@ -363,6 +363,13 @@ public function __construct() {
             require_once( get_template_directory() . '/dt-core/core-endpoints.php' );
             new Disciple_Tools_Core_Endpoints();
 
+            /**
+             * dt-search
+             */
+            require_once( get_template_directory() . '/dt-search/search-endpoints.php' );
+            $this->endpoints['advanced_search'] = new DT_Search_Endpoints();
+            require_once( get_template_directory() . '/dt-search/search-posts.php' );
```

Comment:
Both of these can be included in the existing /dt-posts/dt-posts-endpoints.php 
and /dt-posts/dt-posts.php